module-services/service-audio/ServiceAudio.cpp

```diff
@@ -717,6 +716,7 @@ std::string ServiceAudio::getSetting(const Setting &setting,
             targetPlayback = PlaybackType::CallRingtone;
         }
         else {
+            const std::string defaultValue = {};
             return defaultValue;
```

Comment:
this one was for clarity - it's easier to read then

Reply:
Ok, I get it; I'll restore it to the previous version if you prefer, this is such a subtle change that I doubt if it's gonna affect the performance to any measurable degree :P

Reply:
I think that this one will look better when reverted - let's stay with one style :P

Reply:
Done ;)